kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/UserTaskServiceBase.java

```diff
@@ -258,12 +256,17 @@ public void setDescription(String containerId, Number taskId, String description
     }
 
     public String saveContent(String containerId, Number taskId, String payload, String marshallingType) {
+        return saveContent(containerId, null, taskId, payload, marshallingType);
+    }
+
+    public String saveContent(String containerId, String userId, Number taskId, String payload, String marshallingType) {
+        userId = getUser(userId);
         containerId = context.getContainerId(containerId, new ByTaskIdContainerLocator(taskId.longValue()));
         logger.debug("About to unmarshal task content parameters from payload: '{}'", payload);
         Map<String, Object> parameters = marshallerHelper.unmarshal(containerId, payload, marshallingType, Map.class);
 
         logger.debug("About to set content of a task with id '{}' with data {}", taskId, parameters);
-        Long contentId = userTaskService.saveContent(containerId, taskId.longValue(), parameters);
+        Long contentId = userTaskService.saveContentFromUser(taskId.longValue(), userId, parameters);
```

Comment:
Not really sure whether we need to pass "containerId" as it was done before? Why it is not required now from a functional point of view?

Reply:
not really needed when you have the task id